packages/makeup-listbox-button/src/index.js

```diff
@@ -47,6 +51,12 @@ module.exports = class {
             this._observeMutations();
             this._observeEvents();
         }
+
+        if (this._buttonFloatingLabel) {
```

Comment:
Minor quibble, but can we maintain the the 'El' and 'Selector' postfix naming convention on variables.

Reply:
Done